tests/unit/maxmemory.tcl

```diff
@@ -53,33 +57,21 @@ start_server {tags {"maxmemory" "external:skip"}} {
                 set rr [redis_deferring_client]
                 lappend clients $rr
             }
-
-            # Freeze the server so output buffers will be filled in one event loop when we un-freeze after sending mgets
-            exec kill -SIGSTOP $server_pid
-            for {set j 0} {$j < 5} {incr j} {
-                foreach rr $clients {
-                    $rr mget 1
-                    $rr flush
-                }
-            }
-            # Unfreeze server
-            exec kill -SIGCONT $server_pid
             
-
-            for {set j 0} {$j < 5} {incr j} {
+            set t [clock seconds]
+            while {![check_test $client_eviction] && [expr [clock seconds] - $t] < 20} {
```

Comment:
let's put some comment before this code block that explains it.
something like "keep generating client output buffers until we can observe some effect on keys / client eviction"

Reply:
:+1: